sirepo/package_data/static/js/warpvnd.js

```diff
@@ -571,6 +570,44 @@ SIREPO.app.directive('conductorGrid', function(appState, panelState, plotting) {
                     })
                     .attr('height', function(d) { return yAxisScale(d.y) - yAxisScale(d.y + d.height); })
                     .call(drag);
+                d3.select('.plot-viewport').selectAll('.warpvnd-shape')
+                    .append('title').text(function(d) {
+                        return doesShapeCrossGridLine(d) ? '' : '⚠️ Conductor does not cross a warp grid line and will be ignored';
+                    });
+            }
+
+            function doesShapeCrossGridLine(shape) {
+                var numX = appState.models.simulationGrid.num_x;  // number of vertical cells
+                var halfChannel = toMicron(appState.models.simulationGrid.channel_width/2.0);
+                var cellHeight = toMicron(appState.models.simulationGrid.channel_width / numX);  // height of one cell
+                var numZ = appState.models.simulationGrid.num_z;  // number of horizontal cells
+                var cellWidth = toMicron(appState.models.simulationGrid.plate_spacing / numZ);  // width of one cell
+                if( cellHeight == 0 || cellWidth == 0 ) {  // pathological?
+                    return true;
+                }
+                else {
+                    if( shape.height >= cellHeight || shape.width >= cellWidth ) {  // shape always crosses grid line if big enough
```

Comment:
Looks good to me. Didn't check the math. I'll leave that to @moellep. I'll just comment on the insubstantial stuff. :)